be/src/storage/rowset/column_reader.h

```diff
@@ -208,6 +208,7 @@ class ColumnReader {
 
     using SubReaderList = std::vector<std::unique_ptr<ColumnReader>>;
     std::unique_ptr<SubReaderList> _sub_readers;
+    std::vector<std::string> _map_keys_name;
```

Comment:
Use the id to replace the column instead in the internal storage when reading.

Reply:
we would change it when reading flatten columns later.